content/05.treat.md

```diff
@@ -1,430 +1,208 @@
-of directed graphs in which one atom is selected as the root node. Graph
-convolutions on undirected molecular graphs have eliminated the need to
-enumerate artificially directed graphs, learning feature vectors for atoms that
-are a function of the properties of neighboring atoms and local regions on the
-molecular graph [@tag:Kearnes2016_graph_conv; @tag:AltaeTran2016_one_shot].
-
-Advances in chemical representation learning have also enabled new strategies
-for learning chemical-chemical similarity functions.  Altae-Tran et al.
-developed a one-shot learning network [@tag:AltaeTran2016_one_shot] to address
-the reality that most practical chemical screening studies are unable to provide
-the thousands or millions of training compounds that are needed to train larger
-multi-task networks.  Using graph convolutions to featurize chemicals, the
-network learns an embedding from compounds into a continuous feature space such
-that compounds with similar activities in a set of training tasks have similar
-embeddings.  The approach is evaluated in an extremely challenging setting. The
-embedding is learned from a subset of prediction tasks (e.g. activity assays for
-individual proteins), and only one to ten labeled examples are provided as
-training data on a new task.  On Tox21 targets, even when trained with _one_
-task-specific active compound and _one_ inactive compound, the model is able to
-generalize reasonably well because it has learned an informative embedding
-function from the related tasks. Random forests, which cannot take advantage of
-the related training tasks, trained in the same setting are only slightly better
-than a random classifier.  Despite the success on Tox21, performance on MUV
-datasets, which contains assays designed to be challenging for chemical
-informatics algorithms, is considerably worse.  The authors also demonstrate the
-limitations of transfer learning as embeddings learned from the Tox21 assays
-have little utility for a drug adverse reaction dataset.
-
-These novel, learned chemical feature representations may prove to be essential
-for accurately predicting why some compounds with similar structures yield
-similar target effects and others produce drastically different results.
-Currently, these methods are enticing but do not necessarily outperform classic
-approaches by a large margin.  The neural fingerprints
-[@tag:Duvenaud2015_graph_conv] were narrowly beaten by regression using
-traditional circular fingerprints on a drug efficacy prediction task but were
-superior for predicting solubility or photovoltaic efficiency.  In the original
-study, graph convolutions [@tag:Kearnes2016_graph_conv] performed comparably to
-a multi-task network using standard fingerprints and slightly better than the
-neural fingerprints [@tag:Duvenaud2015_graph_conv] on the drug efficacy task but
-were slightly worse than the influence-relevance voter method on an HIV dataset.
```

Comment:
Removing this period at the end of the sentence should be the only change that shows up, if viewing rich diff or ignoring whitespace.